### PR TITLE
Adds alert level lighting(?)

### PR DESCRIPTION
**Pull request by Torque4607**
Aaaaaaa it worked when I tested it on my private instance, I have no idea how it's going to react. Bear with me, literally my first 'project' in DM.

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -1287,6 +1289,21 @@ obj/machinery/power/apc/proc/autoset(var/cur_state, var/on)
 	update_icon()
 	return 1
 
+	/obj/machinery/power/apc/proc/set_light_color(var/color) //just changes the colors of the ship's lighting.
```
Comment (martinlyra) [inline code/modules/power/apc.dm]: You should remove the spacing/indentation before the function definition here.
```suggestion
/obj/machinery/power/apc/proc/set_light_color(var/color) //just changes the colors of the ship's lighting.
```

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -1287,6 +1289,21 @@ obj/machinery/power/apc/proc/autoset(var/cur_state, var/on)
 	update_icon()
 	return 1
 
+/obj/machinery/power/apc/proc/set_light_color(var/color) //just changes the colors of the ship's lighting.
```
Comment (Naxodile) [inline code/modules/power/apc.dm:1292]: Personally, I would have this proc take the `/decl/security_level` itself as an argument. Then, checking for nulls to reset, have `lightmode` just set to `security_level_lightmode`.

**Comment by Glloyd** [comment]
@Torque4607 Any movement on this?

**Comment by Andromeda-K22** [comment]
I should note that Torque *probably* isn't aware of how to make too many technical changes, as far as I know.

**Pull request closed without merging**